packages/components/src/progress-bar/README.md

```diff
@@ -0,0 +1,32 @@
+# ProgressBar
+
+<div class="callout callout-alert">
+This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.
+</div>
+
+A simple horizontal progress bar component.
+
+Supports two modes: determinate and indeterminate. A progress bar is determinate when a specific progress value has been specified (from 0 to 100), and indeterminate when a value hasn't been specified.
+
+### Props
+
+The component accepts the following props:
+
+#### `value`: `number`
+
+The progress value, a number from 0 to 100.
+If a `value` is not specified, the progress bar will be considered indeterminate.
+
+-   Required: No
+
+##### `className`: `string`
+
+A CSS class to apply to the underlying `div` element, serving as a progress bar track.
+
+- Required: No
+
+##### `id`: `string`
+
+The HTML `id` of the `progress` element. This is necessary to be able to accessibly associate the label with that element.
```

Comment:
```suggestion
Specify a custom HTML `id` for the `progress` element. This is necessary when you want to accessibly associate a custom `label` with that element.
```

I think we can be a bit more clear that this is optional, and only for specific custom use cases.

On the other hand, I [checked](https://codesandbox.io/s/nice-https-f22hyt?file=/src/App.js) what the labeling priority is when there are competing labels, and it seems to be `aria-labelledby` > `aria-label` > `<label for="my-id">`. Therefore, if a consumer wants to do a `<label for="my-progress-bar">`, they're also going to have to remove the default `aria-label` prop, which is non-obvious 🤔 It would be more straightforward to use `aria-labelledby` or `aria-label` if they want to override the label. In this case, we probably shouldn't be calling out the `id` prop explicitly in our docs, as it'd be highlighting a non-recommended approach? (It will just be silently forwarded as part of the rest props.)

Reply:
Since I removed the `id` completely and it's now part of the props we pass down to the underlying `progress` element, this is no longer relevant.